src/librustc/session/mod.rs

```diff
@@ -756,7 +756,7 @@ pub fn build_session_(sopts: config::Options,
         dependency_formats: RefCell::new(FxHashMap()),
         crate_disambiguator: RefCell::new(None),
         features: RefCell::new(feature_gate::Features::new()),
-        recursion_limit: Cell::new(64),
+        recursion_limit: Cell::new(128),
```

Comment:
is this still needed with `move_constraint`?

Reply:
I don't think so. I reverted the changes to the recursion limit.